README.md

```diff
@@ -25,15 +25,6 @@ OPTIONAL SECTIONS:
 After you're finished please remove all the comments and instructions!
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/223779015-77107dea-38ab-40b0-ad71-501c503e8ad0.png)

**[Optional]**

- [ ] It would be nice if you remove all the section descriptions and instructions in your readme file because these instructions are just for you and for no one else so these are unnecessary for the viewers of your repo and also it is making your readme file look not clean 😨 Kindly remove it 😊

Reply:
Implemented